scrapers_next/vt/committees.py

```diff
@@ -0,0 +1,73 @@
+from lxml.html import fromstring
+from lxml.html.clean import Cleaner
+from lxml.etree import ParserError
+from spatula import JsonListPage, URL, SkipItem
+from openstates.models import ScrapeCommittee
+
+
+class CommitteeList(JsonListPage):
+    source = URL(
+        "https://legislature.vermont.gov/committee/loadList/2024/",
+        timeout=10,
+    )
+
+    def process_page(self):
+        for each_committee in self.response.json()["data"]:
+            # name
+            name = each_committee["LongName"]
+
+            # chamber
+            if "house" in name.lower():
+                chamber = "lower"
+            elif "senate" in name.lower():
+                chamber = "upper"
+            else:
+                chamber = "legislature"
+
+            # classification & parent check
+            classification = "committee"
+            parent = None
+            if "subcommittee" in name.lower():
+                classification = "subcommittee"
+                parent = name.lower().split()[-1].replace(")", "")
+
+            # extracting members
+            members_html = each_committee["Members"].strip()
+            members_html = members_html.replace("</br>", "\n")
+            try:
+                doc = fromstring(members_html)
+            except ParserError:
+                SkipItem(f"No membership data found for: {name}")
+                continue
+
+            cleaner = Cleaner(allow_tags=[""], remove_unknown_tags=False)
+            members_list = cleaner.clean_html(doc).text_content()
+
+            com = ScrapeCommittee(
+                name=name,
+                chamber=chamber,
+                parent=parent,
+                classification=classification,
+            )
+
+            for member in members_list.splitlines():
+                if not member.strip():
+                    SkipItem(f"No membership data found for: {name}")
+                    continue
+                try:
+                    name, role = member.split(",")
+                except ValueError:
+                    name = member
+                    role = "Member"
+                com.add_member(name.strip().title(), role=role.strip().title())
+
+            # additional check if no member found
+            if not com.members:
+                SkipItem(f"No membership data found for: {name}")
+                continue
+
```

Comment:
In looking at the source JSON just now, I noticed that there are two potentially useful fields involving URLs for each committee: `"WebSiteURL"` and `"StreamingURL"`.

Could you add a `com.add_link()` method call for each case to store the respective values if they do exist?
- Example for the website url:
```
if each_committee.get("WebSiteURL"):
    com.add_link(each_committee["WebSiteURL"], note="Committee web page")
```

And same for the `"StreamingURL"` key, with the note reading "Committee streaming page".